packages/libs/react-ui/src/components/Grid/GridItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { GridItemClass } from './Grid.css';
+
+import React, { CSSProperties, FC, ReactNode } from 'react';
+
+export type ColSpanType = number | 'auto';
+
+export interface IGridItemProps {
+  children?: ReactNode;
+  colStart?: ColSpanType;
+  colEnd?: ColSpanType;
+  rowStart?: ColSpanType;
+  rowEnd?: ColSpanType;
+  area?: CSSProperties['gridArea'];
+}
+
+const GridItem: FC<IGridItemProps> = ({
+  children,
+  colStart,
+  colEnd,
+  rowStart,
+  rowEnd,
+  area,
+}) => {
+  const styles =
+    area === undefined
+      ? {
+          gridColumnStart: colStart,
+          gridColumnEnd: colEnd,
+          gridRowStart: rowStart,
+          gridRowEnd: rowEnd,
+        }
+      : {
+          gridArea: area,
+        };
```

Comment:
Is there a reason why you opted to change the way this is implemented from the previous grid component?

Reply:
The existing implementation was causing issues for when one set of variables was defined (col and row starts/ends) but the other wasn't (gridarea).